src/client/css/partials/notFound.css

```diff
@@ -0,0 +1,24 @@
+.not-found {
+  background-image: url('/images/error-bg.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: var(--padding-xl);
+}
+
+.not-found h1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: var(--padding-sm);
+}
+
+.error-code {
+  font: var(--h3-font);
+  font-weight: bold;
+  font-size: 10rem;
+  color: var(--purple-60);
+  text-shadow: 0.75rem 0 0 var(--purple-20);
+}
```

Comment:
“No newline at end of file”. :)

Reply:
I think this was the same story of VSCode automatically removing it, but it works now: af46b139